scripts/metadata_comparison/metadata_comparison/extractor.py

```diff
@@ -93,7 +93,7 @@ def upload_workflow_metadata_json(bucket_name: str,
                                   raw_workflow_metadata: bytes,
                                   workflow_gcs_base_path: str,
                                   gcs_storage_client: storage.Client) -> None:
-    workflow_gcs_metadata_upload_path = f'{workflow_gcs_base_path}/metadata.json'
+    workflow_gcs_metadata_upload_path = f'{workflow_gcs_base_path}/workflow.json'
```

Comment:
🤮 

Is a minor cosmetic correction really worth the pain of endless "file not found" exceptions?

Reply:
Not cosmetic, I made this change *because of* "file not found" exceptions. 🙂 The extractor and digester were not in agreement about what this file is named.

Reply:
Ok, so I guess in that case I... uh... retract my vomit?